packages/react-components/react-nav-preview/stories/src/NavDrawer/NavDrawerSize.stories.tsx

```diff
@@ -178,6 +198,20 @@ export const NavDrawerSize = (props: Partial<NavDrawerProps>) => {
             <Radio value="medium" label="Medium" />
             <Radio value="small" label="Small" />
           </RadioGroup>
+          <Label id={linkLabelId}>Links</Label>
+          <Switch
+            checked={enabledLinks}
+            onChange={(_, data) => setEnabledLinks(data.checked as boolean)}
```

Comment:
if the cast is due to possibly being undefined, I think it might be better to do this:
```suggestion
            onChange={(_, data) => setEnabledLinks(!!data.checked)}
```